src/comparisons/elements/create_elements/es6.js

```diff
@@ -0,0 +1,7 @@
+function generateElements(html) {
+  const template = document.createElement('template');
+  template.innerHTML = html.trim();
+  return template.content.firstChild;
```

Comment:
It looks like jQuery is return an HTMLCollection here, so this would be more appropriate as:
```suggestion
  return template.content.children;
```

Reply:
The IE8 example returns `firstChild` unless there are multiple children. Which is more correct?

https://github.com/HubSpot/youmightnotneedjquery/blob/4af243d4a65599814301f659b83539a78caf488b/src/comparisons/elements/create_elements/ie8.js#L7-L11

Reply:
The HTMLCollection is more correct. That IE8 example is actually wrong here:
https://codepen.io/abbondanzo/pen/GRdRoRY

Reply:
Ok, propagated the change to that file as well.